jquery.mockjax/jquery.mockjax.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for jQuery Mockjax 1.5
+// Project: https://github.com/jakerella/jquery-mockjax
+// Definitions by: Laszlo Jakab <https://github.com/laszlojakab/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+///<reference path="../jquery/jquery.d.ts" />
+
+interface MockJaxSettings {
+    url?: string;
+    data?: any;
+    dataType?: string;
+    type?: string;
+    headers?: any;
+    status?: number;
+    statusText?: string;
+    responseTime?: number;
+    isTimeout?: boolean;
+    contentType?: string;
+    response?: (settings: any) => void;
+    responseText?: string;
+    responseXml?: string;
+    proxy?: string;
+    lastModified?: string;
+    etag?: string;
+}
+
+interface MockJaxStatic {
+    (options: MockJaxSettings): number;
+    mockedAjaxCalls(): any; // TODO: any?
+    unfiredHandlers(): any; // TODO: any?
+}
+
+interface JQueryStatic {
+    mockjax: MockJaxStatic;
+    mockjaxClear(id?: number): any;
```

Comment:
This is a deprecated function and will be removed in v2.0.0! You should use `$.mockjax.clear()` instead.